string gps_flag
string direction
float64 gps_heading
float64 imu_heading